modules/core/src/core/shaderlib/lighting/lighting.glsl.js

```diff
@@ -21,14 +21,15 @@
 // lighting
 
 export default `\
-#define NUM_OF_LIGHTS 2
+#define MAX_NUM_OF_LIGHTS 16
```

Comment:
Why so many, so much looping? Would 5 be sufficient?

Reply:
This is the number we stated in the documentation. Totally arbitrary.

Reply:
Should we change the docs then? I am not sure about perf impact but until we measure I'd rather be safe than sorry. Don't think anyone is going to complain if we cap at 5.